components/DownloadPopup.jsx

```diff
@@ -11,9 +11,20 @@ export default function DownloadPopup({ isOpen }) {
   return (
     <>
       {/* <div className="download-overlay w-full h-full fixed bg-gray-800 top-0 left-0 opacity-50 z-100"></div> */}
-      <div className="download fixed w-80 bg-gray-100 p-10 h-40 top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2">
-        <div className="download__container"></div>
+      <div className='download fixed w-80 bg-gray-100 p-10 h-40 top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2'>
+        <div className='download__container mx-3'>
+          <div className='flex flex-row translate-y-1/3 justify-center align-center space-x-3'>
+            {result.map((dInfo, index) => (
+              <button
```

Comment:
Add width and height to `a href`, not the button

Reply:
oh noh. why scold my boi kst